### DISCUSSION TITLE
GrowingIO Design 2021 Roadmap

### DISCUSSION THREAD

**Discussion by jack0pan** [growingio/gio-design#622] [Ideas]
## gio-design 拆分

移除 Lerna，`Changelog` 生成使用 `conventional-changelog` 工具。

- packages/compoents --> gio-design
- packages/icons --> gio-design-icons
- packages/icons -->  gio-design-tokens

## List 列表组件重构

List 组件只实现自有样式，不支持其他组件的样式。要支持的组件太多，导致 List 本身臃肿，不好维护和迭代。

自有样式：
列表项
![image](https://user-images.githubusercontent.com/3879776/102859975-ed63e800-4467-11eb-9e48-76433e6008fb.png)

可拖拽：
![image](https://user-images.githubusercontent.com/3879776/102860043-09678980-4468-11eb-8837-d5ebb5573082.png)

**Comment by jack0pan** [comment]
## GrowingIO Design Icons

参考 [Material Design](https://material.io/resources/icons/?style=baseline)，图标分为三种类型：

- Outlined 线框类型
- Filled 填充类型
- TowTone 双色类型

**Reply by gavin-hao** [reply]
是否可以支持类似antd的自定义图标？

**Reply by jack0pan** [reply]
暂不考虑，有需求再说。
